web/src/pages/Cases/CaseDetails/Voting/VotesDetails/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { CustomAccordion } from "@kleros/ui-components-library";
+import React from "react";
+import styled from "styled-components";
+import { Answer } from "context/NewDisputeContext";
+import { DrawnJuror } from "utils/getDrawnJurorsWithCount";
+import InfoCard from "components/InfoCard";
+import AccordionTitle from "./AccordionTitle";
+import { responsiveSize } from "styles/responsiveSize";
+import { getVoteChoice } from "utils/getVoteChoice";
+import { isUndefined } from "utils/index";
+
+const StyledAccordion = styled(CustomAccordion)`
+  width: 100%;
+  > * > button {
+    justify-content: unset;
+    padding: 11.5px ${responsiveSize(8, 18)} !important;
+    background-color: ${({ theme }) => theme.whiteBackground} !important;
+    border: 1px solid ${({ theme }) => theme.stroke} !important;
+    > svg {
+      fill: ${({ theme }) => theme.primaryText} !important;
+    }
+  }
+  //adds padding to body container
+  > * > div > div {
+    padding: 8px ${responsiveSize(8, 16)};
+  }
+`;
+const JustificationContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  > p {
+    margin: 0px;
+  }
+`;
+
+const StyledLabel = styled.label`
+  color: ${({ theme }) => theme.primaryText};
+`;
+
+const AccordionContent: React.FC<{
+  choice?: string;
+  answers: Answer[];
+  justification: string;
+}> = ({ justification, choice, answers }) => (
+  <JustificationContainer>
+    {!isUndefined(choice) ? (
+      <StyledLabel>
+        Voted : <small>{getVoteChoice(parseInt(choice), answers)}</small>
```

Comment:
I think I am with @kemuru on this, let's remove the `Voted :` and leave only the choice
```suggestion
        <small>{getVoteChoice(parseInt(choice), answers)}</small>
```

Reply:
The "Voted :" here is the one in extended accordion, on the title "Voted :" has already been removed.

Reply:
<img width="998" alt="Screenshot 2024-01-23 at 23 03 19" src="https://github.com/kleros/kleros-v2/assets/51452848/1feaabcb-ff3c-4d3a-b4c0-2e7cf71693a9">
 final look